script/ebooks

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+// Generate EPUB and MOBI ebooks
+
+const fs = require('fs');
+const path = require('path');
+const exec = require('child_process').exec;
+const fastmatter = require('fastmatter');
+const mkdirp = require('mkdirp');
+const slug = require('slug');
+const unified = require('unified');
+const parse = require('remark-parse');
+const stringify = require('remark-stringify');
+const visit = require('unist-util-visit');
+const download = require('download');
+const ebrew = require('ebrew');
+
+const processor = unified()
+  .use(parse)
+  .use(replaceAsides)
+  .use(replaceCheckboxes)
+  .use(fixImages)
+  .use(downloadImages)
+  .use(stringify);
+
+buildBooks([
+  '_articles/how-to-contribute.md',
+  '_articles/starting-a-project.md',
+  '_articles/finding-users.md',
+  '_articles/building-community.md',
+  '_articles/best-practices.md',
+  '_articles/leadership-and-governance.md',
+  '_articles/getting-paid.md',
+  '_articles/code-of-conduct.md',
+  '_articles/metrics.md',
+  '_articles/legal.md',
+]);
```

Comment:
Could we read the `_articles` directory here, so we don’t need to update this list?